### ISSUE TITLE
Screen Reader: Workspaces: Adjacent checkbox and button performs same action, focused and announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76955]
## Action Performed:
Prerequisites:
- the user is signed in

1. Using Chrome + JAWS, open the URL https://new.expensify.com/
2. Navigate using the Tab key to the Workspaces section.
3. Move focus to the More options icon (three vertical dots) > Duplicate Workspace
4. Navigate using the tab key to the Next button and press Enter to activate it.
5. Navigate using the tab key to the Select all checkbox and observe the screen reader announcement.

## Expected Result:
There should be a single interactive control for the “Select all” functionality. If visual design requires both a checkbox and a label, the label should be programmatically associated with the checkbox using appropriate ARIA attributes (aria-labelledby, for + id, etc.), so they are announced as a single control.

## Actual Result:
The adjacent checkbox and button labeled “Select all” perform the same action but are announced and focused separately by the screen reader. When navigating using JAWS, both elements receive focus individually and are announced as separate controls (“Select all checkbox” and “Select all button”), even though both trigger the same functionality. This creates confusion for screen reader users and results in redundant controls.

HTML:
<div aria-checked="true" aria-label="Select all" role="checkbox" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style="justify-content: center; align-items: center; border-radius: 6px;"><div class="css-g5y9jx r-18og6j9 r-8drixv" style="height: 20px; width: 20px; border-width: 2px; justify-content: center; align-items: center; border-radius: 4px; margin: 2px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="14" height="14" hovered="false" pressed="false" fill="#E7ECE9"><path d="M17.2 3.4c.9.7 1.1 1.9.4 2.8L8.9 18l-6.3-5.7c-.8-.7-.9-2-.1-2.8.7-.8 2-.9 2.8-.1l3 2.7 6.1-8.2c.6-1 1.9-1.2 2.8-.5" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div></div>

<button aria-label="Select all" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div dir="auto" class="css-146c3p1" data-element="Text" data-source-file="WorkspaceDuplicateSelectFeaturesForm.tsx" style="color: rgb(175, 187, 176); font-size: 13px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; padding-right: 12px; padding-left: 12px;">Select all</div></button>

## Area issue was found in:
Workspace

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Remove the redundant button element.
https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/

## Environment(s)
Windows Windows 11 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/e5926cf2-1718-4a26-a4e6-022deaa2933b

<img width="1919" height="972" alt="Image" src="https://github.com/user-attachments/assets/62c62cbc-6e61-411b-b1ce-6ac9aa7fa6de" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6999540
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
done in https://github.com/Expensify/App/issues/79232
